wiki/Ranking_Criteria/en.md

```diff
@@ -69,24 +81,27 @@ Glossary
+-   **Single-mode mapsets must include a reasonable spread of at least two difficulties.** The lowest difficulty cannot be harder than a Normal and it must comply with its respective mode’s difficulty-specific ranking criteria.
+-   **Hybrid mapsets without osu!standard difficulties must include a reasonable spread of at least two difficulties per mode.** The lowest difficulty of each mode cannot be harder than a Normal and it must comply with its respective mode’s difficulty-specific ranking criteria.
+-   **If a hybrid mapset includes osu!standard difficulties...**
+    -   **A reasonable spread of at least two osu!standard difficulties must be included.** The lowest difficulty cannot be harder than a Normal and it must comply with its respective mode’s difficulty-specific ranking criteria.
+    -   **Converted difficulties must form a reasonable spread.** For example, a mapset with Easy and Normal osu!standard difficulties and an Insane osu!catch difficulty is not permitted. One or more additional difficulties may need to be added to fill the gap.
+    -   **Any two or more osu!taiko or osu!mania difficulties must be arranged in a reasonable spread.** The lowest difficulty cannot be harder than a Hard.
+    -   **One or more osu!catch difficulties may be included.** The lowest difficulty cannot be harder than an Insane.
+-   **Mapsets must have a minimum drain time of 30 seconds.** This ensures each ranked map has a practical play-time.
+-   **Marathons must have a minimum drain time of 5 minutes.** This excludes especially long mapsets from requiring a spread of difficulties.
+-   **Excluding a mapset’s hardest difficulty, a difficulty’s name must accurately indicate its level of difficulty.** Conventional difficulty names vary between modes, but any set of clearly progressive difficulty names can be alternatively used. Additionally, a mapset’s hardest difficulty should not use a name misrepresentative of its difficulty.
+-   **A difficulty’s name must be unrelated to a username.** Guest difficulties, however, may indicate possession with its mappers’ username or nickname. (e.g. Guest Mapper’s Insane). Words that happen to be usernames are acceptable within difficulty names as long as they relate to the song.
+-   **Additionally, a mapset host cannot indicate possession in a difficulty’s name.** (e.g. Mapset Host’s Insane). Conflicts caused by mapping multiple songs with the same metadata are the only exception.
+-   **A mapset host must have equal or more drain time mapped than any guest difficulty mappers.** This is to provide credit where credit is due.
+-   **A mapset host and guest mappers can make changes to their respective difficulties as they wish.** If there is a disagreement between both, the mapset host must delete the guest contribution upon request. If a guest mapper cannot be contacted for a month, they will be assumed to agree with any changes. 
```

Comment:
"the mapset host _may_ delete", this is an option that can be resolved.

Reply:
if conflicts are resolved, there isn't a disagreement anymore i think. "must" was used because "may" makes it sound like even if gd mapper doesn't approve of something host did to their gd, host can still keep it anyway.